main.py

```diff
@@ -0,0 +1,54 @@
+from utils.objects.metadata_object import MetaDataObject
```

Comment:
I think its a good idea to have a script for quick local testing but for the demonstration i think the notebook is the better way to go for now as we can explain more there.